### PR TITLE
Updates haul-integrate.sh to allow receiving a custom script source

### PR DESCRIPTION
**Pull request by sky-franciscogoncalves**
This PR updates `haul-integrate.sh` script to allow receiving a custom script source. This is particularly useful when using Haul together with libraries like [React Native Schemes Manager](https://github.com/blargity/react-native-schemes-manager) that provide their own `react-native-xcode.sh` script.

To make use of this feature, users should update their `Integrate Haul with React Native`build phase to:

```sh
export SCRIPT_SRC="react-native-schemes-manager/lib"
../node_modules/haul/src/utils/haul-integrate.sh
```

However, if the user does not provide a `SCRIPT_SRC`, Haul will keep its previous behaviour.

Please let me know if I can improve this PR in anyway.

### CODE REVIEW

**Comment by sky-franciscogoncalves** [comment]
@thymikee Thanks for your feedback. I've added the documentation and merged it with next.

File: package.json [hunk 1]
```diff
@@ -164,4 +164,4 @@
       "**/*.test.js"
     ]
   }
-}
+}
```
Comment (thymikee) [inline package.json]: Can you bring the newline back?

File: src/server/middleware/symbolicateMiddleware.js [hunk 2]
```diff
@@ -19,8 +19,8 @@
 import type { $Request, Middleware } from 'express';
 import type {
   ReactNativeStackFrame,
-  ReactNativeStack,
-  Platform,
+    ReactNativeStack,
+    Platform,
```
Comment (thymikee) [inline src/server/middleware/symbolicateMiddleware.js]: Mind fixing the indentation? :)

**Comment by jukben** [comment]
screen shot 2018-07-10 at 18 23 49
This doesn't help? We have just killed haul-integrate.sh.

**Comment by jukben** [comment]
I guess we can close this PR due to the fact we have removed haul-integrate at all and we are using ENV approach. @thymikee

**Pull request closed without merging**